package.json

```diff
@@ -11,6 +11,8 @@
     "compile": "forge compile",
     "build": "forge build",
     "test": "forge test",
+    "test:unit": "forge test --no-match-contract 'Integration'",
```

Comment:
ahh just noticed this should be `Unit`

Reply:
Ohh, should have commented on the PR: it's a `no-match-contract` to include the strategies as well (as they don't start with Unit but should be included imo) - should I exclude them from the unit tests?

Reply:
ahh you are right, lets leave it as is then